utils/emoticons.test.jsx

```diff
@@ -5,6 +5,41 @@ import * as Emoticons from 'utils/emoticons.jsx';
 
 describe('Emoticons', () => {
     describe('handleEmoticons', () => {
+        // test emoticon patterns
+        const emoticonPatterns = {
+            slightly_smiling_face: [':)', ':-)'],
+            wink: [';)', ';-)'],
+            open_mouth: [':o'],
+            scream: [':-o'],
+            smirk: [':]', ':-]'],
+            smile: [':D', ':-D'],
+            stuck_out_tongue_closed_eyes: ['x-d'],
+            stuck_out_tongue: [':p', ':-p'],
+            rage: [':@', ':-@'],
+            slightly_frowning_face: [':(', ':-('],
+            cry: [':`(', ':\'(', ':’('],
+            confused: [':/', ':-/'],
+            confounded: [':s', ':-s'],
+            neutral_face: [':|', ':-|'],
+            flushed: [':$', ':-$'],
+            mask: [':-x'],
+            heart: ['<3', '&lt;3'],
+            broken_heart: ['</3', '&lt;/3'],
+            thumbsup: [':+1:'],
+            thumbsdown: [':-1:'],
+        };
+        for (const key in emoticonPatterns) {
+            if (emoticonPatterns.hasOwnProperty(key)) {
```

Comment:
Force of habit! :)

To prevent something like this:
```
let test = {};
for(const key in test) {
    console.log(key);
}
Object.prototype.fiz = 'buz';
```

That said, it's not really necessary in this context so I'll change to using the below.